Makefile

```diff
@@ -26,3 +27,12 @@ tag:
 	@git-chglog --output CHANGELOG.md
 	@git commit -m 'Update CHANGELOG.md' -- CHANGELOG.md
 	@git tag -f "${TAG}"
+
+.PHONY: docker-build
+docker-build: test ## Build docker image with the manager.
+	docker build -t ${IMG} .
+
+.PHONY: docker-push
+docker-push: ## Push docker image with the manager.
+	docker push ${IMG}
```

Comment:
nitpicking, do we really need these tasks? I'm concerned someone might accidentally push the `policy-server:latest` to GH from his local machine...

Reply:
Agree here. It's reassuring to use `policy-server:latest` and know that it comes from main and tests passed. Maybe drop the `docker-push` target? The `docker-build` is useful for local dev.

Reply:
I used these targets to build and publish the container image later used my tests scripts. But I understand the concerns. Let me change this.